Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalIsolatedBunchFilter_Output_cff.py

```diff
@@ -0,0 +1,22 @@
+import FWCore.ParameterSet.Config as cms
+
+# output block for alcastream HCAL IsolatedBunch
+# output module 
+#  module alcastreamHcalIsolatedBunchOutput = PoolOutputModule
+
+OutALCARECOHcalCalIsolatedBunchFilter_noDrop = cms.PSet(
+    # use this in case of filter available
+    SelectEvents = cms.untracked.PSet(
+        SelectEvents = cms.vstring('pathALCARECOHcalCalIsolatedBunchFilter')
+    ),
+    outputCommands = cms.untracked.vstring( 
+        'keep *_hbhereco_*_*',
+        'keep *_ecalRecHit_*_*',
+        'keep *_hltTriggerSummaryAOD_*_*',
+        'keep *_TriggerResults_*_*',
+        )
```

Comment:
@bsunanda please explicit the types of the products.
Also, there was mention in [presentation](https://indico.cern.ch/event/558794/contributions/2254652/attachments/1315042/1969666/Phase1_13.pdf) that RAW products would have been kept, but I don't see them. Please clarify